### ISSUE TITLE
Status mail date/time issue

### ISSUE THREAD

**Issue by DrCarsonBeckett** [virtualmin/virtualmin-gpl#169]
Very often, but not everytime, when I receive a status mail (like after a backup was done) the date and time are not displayed correctly:
`Aktion durchgeführt um: <x-md-d>1590655833;0</x-md-d> <x-md-t>1590655833</x-md-t>`

10 Minutes later, it shows it correctly:
`Aktion durchgeführt um: 28/Mai/2020 10:59`

I am not sure why it happens not everytime, but I still think it should be looked into.
I can provide more details about the system/etc, when its necessary.

**Comment by DrCarsonBeckett** [comment]
To clarify, with 10 minutes later I meant a second mail which I received ten minutes later.

**Comment by iliarostovtsev** [comment]
Hi,

That should no longer be an issue with the latest software packages. What is your **Webmin**, **Virtualmin** and **Authentic Theme** versions?

**Comment by DrCarsonBeckett** [comment]
Webmin-Version | 1.942
Usermin-Version | 1.791
Virtualmin Version | 6.09
Authentic theme version | 19.47

Latest versions for me without updating directly from the repo (since the CP gives you a warning about it).

**Comment by iliarostovtsev** [comment]
> Very often, but not everytime, when I receive a status mail (like after a backup was done) the date and time are not displayed correctly.

Perhaps, it happens when the action is triggered from UI?

**Comment by DrCarsonBeckett** [comment]
Just gone through the latest logging/status mails and sadly it works fine most of the time for UI triggered mails, but not for all. Meaning, that issue isnt always triggered from the UI.
But form what I can see, it never happens if the mail is triggered from cron or similar systems.

**Comment by jcameron** [comment]
How exactly are you triggering the email from the UI?

**Comment by DrCarsonBeckett** [comment]
I just noticed its properly a webmin function and not virtualmin, sorry for that.
At least the mails about user-related changes are send with a notice that its from webmin.
I enabled that function in webmin -> webmin configuration -> protocoll settings 
For the backup, I enabled the report function during the "scheduled backup" settings.
The packages report, I enabled in the "scheduled upgrades" in checking for new packages.


And I checked the mails again, that behaviour only occured with mails send from webmin.
Today I noticed this 
`ge&#228;ndert`
Which should be:
`geändert`

Sorry for the wrong placement of the ticket.
